src/Staker.sol

```diff
@@ -77,6 +96,15 @@ abstract contract Staker is INotifiableRewardReceiver, Multicall {
   /// @notice Emitted when a reward notifier address is enabled or disabled.
   event RewardNotifierSet(address indexed account, bool isEnabled);
 
+  /// @notice Emitted when a deposit's earning power is changed via bumping.
+  event EarningPowerBumped(
```

Comment:
Do we care about the address of the bumper?

Reply:
Good suggestion, added.